rpmbuild/copr_rpmbuild/helpers.py

```diff
@@ -469,3 +469,20 @@ def package_version(name):
             return pkg_resources.require(name)[0].version
         except pkg_resources.DistributionNotFound:
             return "git"
+
+
+def mock_snippet_for_tags(job_tag_to_mock_snippets, tags):
+    """
+    Return mock snippets as string separated by newlines for a given
+    list of tags.
+    """
+    if not tags or not job_tag_to_mock_snippets:
+        return ""
+
+    tags_set = set(tags)
+    mock_snippets = []
+    for tag, snippet in job_tag_to_mock_snippets.items():
+        if set(tag.split("+")).issubset(tags_set):
```

Comment:
Don't you want to accept list instead?  What if the `+` symbol is part of the label?

Reply:
Check this option:
```
- tags:
    - on_demand_powerful
    - arch_x86_64
  the_snippet: |
    some_multi_
    line_snippet of your choice

Reply:
resolved on slack